_posts/en/newsletters/2024-05-31-newsletter.md

```diff
@@ -0,0 +1,216 @@
+  the feature can be added to Bitcoin.  
+
+## Selected Q&A from Bitcoin Stack Exchange
+
+*[Bitcoin Stack Exchange][bitcoin.se] is one of the first places Optech
+contributors look for answers to their questions---or when we have a
+few spare moments to help curious or confused users.  In
+this monthly feature, we highlight some of the top-voted questions and
+answers posted since our last update.*
+
+{% comment %}<!-- https://bitcoin.stackexchange.com/search?tab=votes&q=created%3a1m..%20is%3aa
+nswer -->{% endcomment %}
+{% assign bse = "https://bitcoin.stackexchange.com/a/" %}
+
+- [What's the smallest possible coinbase transaction / block size?]({{bse}}122951)
+  Antoine Poinsot explains minimum restrictions around the coinbase transaction
+  and concludes the smallest possible valid Bitcoin block at current block heights is 145 bytes.
+
+- [Understanding Script's number encoding, CScriptNum]({{bse}}122939)
+  Antoine Poinsot describes how CScriptNum represents integers in Bitcoin's
+  Script, provides some example encodings, and links to two serialization
+  implementations.
+
+- [Is there a way to make a BTC wallet address public but hide how many BTC it contains?]({{bse}}122786)
+  Vojtěch Strnad points out that [silent payment][topic silent payments] reusable
+  payment addresses allow a public payment identifier to be posted without
+  observers being able to associate transactions paying to it.
+
+- [Testing increased feerates in regtest]({{bse}}122837)
+  In regtest, Ava Chow recommends using Bitcoin Core's test framework and
+  setting `-maxmempool` to a low value and `-datacarriersize` to a high value in
+  order to help simulate high feerate environments.
+
+- [Why is my P2P_V2 peer connected over a v1 connection?]({{bse}}122774)
+  Pieter Wuille hypothesizes that outdated peer addr information was what caused
+  a user to see a peer that supported BIP324 [encrypted transport][topic v2 p2p
+  transport] that was connected with a v1 non-encrypted connection.
+
+- [Does a P2PKH transaction send to the hash of the uncompressed key or the compressed key?]({{bse}}122875)
+  Pieter Wuille notes that both compressed and uncompressed public keys can be
+  used resulting in distinct P2PKH addresses, adding that P2PKH only supports
+  compressed public keys and P2TR uses [x-only public keys][topic X-only public keys].
+
+- [What are different ways to broadcast a block to the Bitcoin network?]({{bse}}122953)
+  Pieter Wuille outlines 4 ways to announce blocks on the P2P network: using
+  [BIP130][], using [BIP152][], unsolicited `block` messages, and the older
+  `inv` / `getdata` / `block` message flow.
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [LND v0.18.0-beta.rc3][] is a release candidate for the next major
+  version of this popular LN node implementation.
```

Comment:
In the meantime, [beta](https://github.com/lightningnetwork/lnd/releases/tag/v0.18.0-beta) was released.

```suggestion
- [LND v0.18.0-beta][] is beta release of the next major
  version of this popular LN node implementation.
```